packages/cli/src/logging/logger.service.ts

```diff
@@ -58,9 +58,10 @@ export class Logger {
 		this.internalLogger = internalLogger;
 	}
 
-	withScope(scope: LogScope) {
+	/** Create a logger that injects the given scopes into its log metadata. */
+	scoped(scopes: LogScope[]) {
```

Comment:
```suggestion
	scoped(scopes: LogScope | LogScope[]) {
	  scopes = Array.isArray(scopes) ? scopes : [scopes];
```
Not necessary, but I think this would be a nicer API.